post-install.yml

```diff
@@ -288,97 +291,127 @@
     tags:
       - "19"
 
   - name: POSTINSTALL 20.0 | Enable the API Certificate
     command: "oc patch apiserver cluster --type=merge -p '{\"spec\":{\"servingCerts\": {\"namedCertificates\": [{\"names\": [\"{{ oc_api_domain }}\"], \"servingCertificate\": {\"name\": \"{{ oc_api_domain }}\"}}]}}}'"
-    register: result
+    # failed_when: "'no change' not in result20.stdout"  # use this when run multiple times
+    register: result20
     tags:
       - "20"
 
   - name: POSTINSTALL 21.0 | Enable Monitoring Persistent Storage
     k8s:
       state: present
-      apply: "yes"
+      namespace: openshift-monitoring
+      # apply: "yes"  # only works with Ansible 2.9 and higher
       src: "./postinstall/cluster-monitoring-config.yml"
     tags:
       - "21"
 
-  - name: POSTINSTALL 22.0 | Enable Cluster Logging
+  - name: POSTINSTALL 22.0 | Enable Cluster Logging (One)
     command: "{{ item }}"
-    register: result
     loop:
       - "oc create -f ./postinstall/logging/eo-namespace.yml"
       - "oc create -f ./postinstall/logging/clo-namespace.yml"
       - "oc create -f ./postinstall/logging/eo-og.yml"
-      - "oc create -f ./postinstall/logging/eo-sub.yml"
       - "oc create -f ./postinstall/logging/eo-rbac.yml -n openshift-operators-redhat"
-      - "oc create -f ./postinstall/logging/cluster-logging-og.yml -n openshift-logging"
       - "oc create -f ./postinstall/logging/cluster-logging-olm.yml -n openshift-logging"
+    register: result220
+    # failed_when: "'already exists' not in result220.stderr"  # use this when run multiple times
+    tags:
+      - "22"
+
+  - name: POSTINSTALL 22.1 | Enable Cluster Logging (Two)
+    command: "{{ item }}"
+    loop:
+      - "oc create -f ./postinstall/logging/eo-sub.yml"
+      - "oc create -f ./postinstall/logging/cluster-logging-og.yml -n openshift-logging"
+    register: result221
     tags:
       - "22"
 
   - name: POSTINSTALL 23.0 | Wait for Cluster CR to appear
     command: "sleep 30"
-    register: result
+    register: result23
     tags:
       - "23"
 
-  - name: POSTINSTALL 24.0 | Create the CRD Logging
+  # - name: POSTINSTALL 24.0 | Install the Cluster Logging Operator
+  #   command: "oc create -f ./postinstall/logging/clo-og.yml"
+  #   register: result240
+  #   failed_when: "'already exists' not in result240.stderr"
+  #   tags:
+  #     - "24"
+
+  # - name: POSTINSTALL 24.1 | Create a Subscription object, channel 4.3
+  #   command: "oc create -f ./postinstall/logging/clo-sub.yml"
+  #   register: result241
+  #   failed_when: "'already exists' not in result241.stderr"
+  #   tags:
+  #     - "24"
+
+    # Possible error when using this on older OCP (e.g. OCP v4.2)
+    # error: unable to recognize "postinstall/logging/cluster-logging-crd.yml": no matches for kind "ClusterLogging" in version "logging.openshift.io/v1"
+  - name: POSTINSTALL 24.2 | Create a Cluster Logging instance
     command: "oc create -f ./postinstall/logging/cluster-logging-crd.yml"
```

Comment:
@waynedovey 

Do you know what could be causing an error `no matches for kind "ClusterLogging" in version "logging.openshift.io/v1"` in here? I was following [Install the Cluster Logging Operator 4.3 using the CLI](https://docs.openshift.com/container-platform/4.3/logging/cluster-logging-deploying.html#cluster-logging-deploy-clo-cli_cluster-logging-deploying).